client/src/components/LogChannel.ts

```diff
@@ -4,14 +4,14 @@ import { OutputChannel, l10n, window } from "vscode";
 
 import { appendLog } from "./LogViewer";
 
-let outputChannel: OutputChannel;
+let log: OutputChannel;
```

Comment:
In my opinion, `outputChannel` is a better name than `log`. When I see `log`, I'm confused is it log strings or anything.

Reply:
fixed in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/a3aff3515f0aebd91ff1b1ee9ffecaaad9a5b3fe